tutorials/change-detection/README.md

```diff
@@ -0,0 +1,164 @@
+      D: True                    # without energy, but with
+      DD: True                   # energy derivatives
+      stack: 1
+
+architecture:
+   name: StackedLSTM
+   params:                       # this experiments relies
+     n_classes: 1                # on one LSTM layer (16 outputs)
+     lstm: [16]                  # and one dense layer.
+     mlp: [16]                   # LSTM is bidirectional
+     bidirectional: 'concat'
+     final_activation: 'sigmoid'
+
+sequences:
+   duration: 3.2                 # this experiments relies
+   step: 0.8                     # on sliding windows of 3.2s
+   balance: 0.05                 # with a step of 0.8s
+
+```
+
+### Training
+
+The following command will train the network using the training set of the `TV` protocol of the ETAPE database. This may take a long time...
+
+```bash
+$ export EXPERIMENT_DIR=tutorials/change-detection
+$ pyannote-change-detection train \       #  
+          ${EXPERIMENT_DIR} \             # <experiment_dir>
+          Etape.SpeakerDiarization.TV     # <database.task.protocol>
+Epoch 1/100
+62464/62464 [==============================] - 171s - loss: 0.1543 - acc: 0.9669   
+Epoch 2/100
+62464/62464 [==============================] - 117s - loss: 0.1375 - acc: 0.9692     
+Epoch 3/100
+62464/62464 [==============================] - 115s - loss: 0.1376 - acc: 0.9691     
+...
+Epoch 50/100
+62464/62464 [==============================] - 112s - loss: 0.0903 - acc: 0.9724  
+...
+
+```
+
+This will create a bunch of files in `TRAIN_DIR` (defined below), including plots showing the accuracy epoch after epoch.
+
+### Evaluation
+Now that the network is trained, we get different models for different epochs. We can evaluate a specific model:
+
+```bash
+$ export TRAIN_DIR=${EXPERIMENT_DIR}/train/Etape.SpeakerDiarization.TV.train
+$ pyannote-change-detection evaluate \
+         --epoch 49 \
+         ${TRAIN_DIR} \               # <train_dir>
+         Etape.SpeakerDiarization.TV  # <database.task.protocol> 
+```
+
+This is the expected output:
+
+```
+threshold purity  coverage
+--------- ------- --------
+0         95.720% 36.603%
+0.0526316 95.526% 49.018%
+0.105263  95.213% 57.660%
+...
+0.526316  92.396% 83.756%
+...
+0.894737  88.155% 91.139%
+0.947368  87.468% 92.046%
+1         86.929% 92.672%
+
+```
+
+
+
+### Testing
+
+We can choose the best model according to the evaluation results and  apply it on the development set of the `TV` protocol of the ETAPE database:
+
+```bash
+$ pyannote-speech-detection apply \
+          --epoch 49 \
+          ${TRAIN_DIR} \               #<train_dir>
+          Etape.SpeakerDiarization.TV  # <database.task.protocol>
+```
+
+this will create a list of files in `APPLY_DIR` (defined below) containing segmentation results.
+
+```bash
+$ export APPLY_DIR=${TRAIN_DIR}/segment/Etape.SpeakerDiarization.TV.development/0.1/
+$ head -n 5 $APPLY_DIR/BFMTV_BFMStory_2011-03-17_175900.0.seg
```

Comment:
Since the `.seg` file format is not standard, you should probably add a short sentence explaining the file format.